placeholderkv.conf

```diff
@@ -150,10 +150,10 @@ tcp-backlog 511
 # Unix socket.
 #
 # Specify the path for the Unix socket that will be used to listen for
-# incoming connections. There is no default, so Redis will not listen
+# incoming connections. There is no default, so the server will not listen
 # on a unix socket when not specified.
 #
-# unixsocket /run/redis.sock
+# unixsocket /run/server.sock
```

Comment:
I don't think we can claim the word "server" globally. In the context of this project yes, but not in the context of everything running on a computer. Here we may want a more unique default.

Reply:
As this path was commented by default, I chose to proceed with `server.sock`.  Maybe add `placeholderkv.sock` and revisit once the name is finalized?